.ebextensions/00-environment.config-dist

```diff
@@ -0,0 +1,80 @@
+option_settings:
+
+  # Dev / stage settings
+  - option_name: DEBUG
+    value: True
+
+  # Logging setup
+  - option_name: DJANGO_LOG_LEVEL
+    value: ERROR
+  - option_name: DJANGO_LOG_MAXBYTES
+    value: 16777216
+  - option_name: DJANGO_LOG_BACKUP_COUNT
+    value: 3
+  - option_name: DJANGO_LOG_FILENAME
+    value: /var/log/django/app.log
+  - option_name: DJANGO_LOG_FORMATTER
+    value: verbose
+
+  # Credentials used to create the initial superuser. Ignored if at least one
+  # superuser is already available.
+  - option_name: INITIAL_ADMIN_USERNAME
+    value: admin
+  - option_name: INITIAL_ADMIN_PASSWORD
+    value: admin
+  - option_name: INITIAL_ADMIN_EMAIL
+    value: lorchard@mozilla.com
+
+  # SECRET_KEY *must* have a unique, secret value
+  - option_name: SECRET_KEY
+    value: YOUR SECRET KEY HERE
+
+  # Expected hostname(s) for the site
+  - option_name: ALLOWED_HOSTS
+    value: ideatown.firefox.com
+
+  # URL to the current version of the Idea Town addon
+  - option_name: ADDON_URL
+    value: http://example.com/idea-town.xpi
```

Comment:
This is just the example file. The real settings are totally filled in differently than this and uploaded to a private S3 bucket